frontend/src/components/pages/Dashboard/index.tsx

```diff
@@ -9,6 +9,8 @@ import AuthContext from "../../../contexts/AuthContext";
 import { Schedule } from "../../../types/SchedulingTypes";
 import DropoffCard from "./components/DropoffCard";
 
+const upcomingWeekLimit = "2";
```

Comment:
```suggestion
const UPCOMING_WEEK_LIMIT = "2";
```
I also think we should move this constant value into a file in `/constants` but up to you